Composer/packages/client/src/components/Modal/AlertDialog.tsx

```diff
@@ -45,9 +32,9 @@ type Props = {
 
 const AlertDialog = (props: Props) => {
   const { setting, onCancel, onConfirm } = props;
-  const { title, subtitle = '', confirmText = 'Ok', style = dialogStyle.normal } = setting;
+  const { title, subtitle = '', confirmText = formatMessage('Ok'), style = dialogStyle.normal } = setting;
   if (!title) {
-    throw new Error('Confirmation modal must have a title');
+    throw new Error(formatMessage('Confirmation modal must have a title'));
```

Comment:
Nit: It can be nice to have a period at the end of error message sentences. When they get combined it makes them easier to read.